pkgs/top-level/python-packages.nix

```diff
@@ -13867,6 +13867,30 @@ in {
     clblas = pkgs.clblas-cuda;
   };
 
+    libnacl = buildPythonPackage rec {
```

Comment:
Done, it will be included in next push. 
There is now only `libnacl = callPackage ../development/python-modules/libnacl/default.nix { };` in `python-packages.nix`